optimize/self-managed/optimize-deployment/version-policy.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: version-policy
 title: "Version policy"
-description: "Learn about the Versioning policy for Camunda Optimize."
+description: "Learn about the versioning policy for Camunda Optimize."
 ---
 
-## Version Policy
-
 Camunda Optimize versions are denoted as X.Y.Z as well as by an optional [pre-release](https://semver.org/spec/v2.0.0.html#spec-item-9) version being either '-alpha-[0-9]' or '-preview-[0-9]'. X is the [major version](https://semver.org/spec/v2.0.0.html#spec-item-4), Y is the [minor version](https://semver.org/spec/v2.0.0.html#spec-item-7), Z is the [patch version](https://semver.org/spec/v2.0.0.html#spec-item-6) as defined by the [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html) specification.
 
-## Release Cadence
+## Release cadence
 
-Camunda Optimize has a new release every month. Every six months, a new minor/major is published, while all other months an alpha and/or patch release are released. The release dates are aligned with the rest of the [Camunda 8 Platform]($docs$/reference/release-policy).
+Camunda Optimize has a new release every month. Every six months, a new minor/major is published, while all other months an alpha and/or patch release are released. The release dates are aligned with the rest of [Camunda 8]($docs$/reference/release-policy).
```

Comment:
Maybe we should rephrase this to keep it a bit more general given that 8.4.0 doesn't fall into this cadence.

Maybe it would be enough to simply link to the C8 page without the description?